scripts/update-proposal-tests.sh

```diff
@@ -18,20 +18,14 @@ fi
 
 function install_wasi_sdk()
 {
-    curl -L -f https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-${WASI_VERSION}/wasi-sdk-${WASI_VERSION}.0-linux.tar.gz --output wasi-sdk.tar.gz
+    curl -L -f https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-${WASI_VERSION}%2Bthreads/wasi-sdk-${WASI_VERSION}.0.threads-linux.tar.gz --output wasi-sdk.tar.gz
```

Comment:
now 20.0 is available. little point to use the prerelease.

Reply:
yeah agree; the PR was raised long before wasi sdk 20 was released; updated the change.